.github/workflows/release.yaml

```diff
@@ -11,6 +11,9 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+
+      - uses: docker/setup-buildx-action@v2
```

Comment:
See https://github.com/docker/setup-buildx-action#usage

I chose not to  include the optional `docker/setup-qemu-action@v2`  action because the linux/arm64 image seems to build fine without it.

Reply:
You mean the `docker/setup-buildx-action@v2` action? I think that's because we explicitly create a builder in the make target (`docker buildx create ..`)

Reply:
You're right. I removed it and the build still succeeded. Thanks.